src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -684,3 +560,239 @@ IN_PROCESS_APPLICATION::FindHighestDotNetVersion(
     // we check FAILED(hr) outside of function
     return hr;
 }
+
+//
+// Runs a standalone appliction.
+// The folder structure looks like this:
+// Application/
+//   hostfxr.dll
+//   Application.exe
+//   Application.dll
+//   etc.
+// We get the full path to hostfxr.dll and Application.dll and run hostfxr_main,
+// passing in Application.dll.
+// Assuming we don't need Application.exe as the dll is the actual application.
+//
+HRESULT
+IN_PROCESS_APPLICATION::GetStandaloneApplicationArguments(
+    STRU* struApplicationDllPath,
+    STRU* struHostfxrPath
+)
+{
+    HRESULT     hr = S_OK;
```

Comment:
I don't even understand this formatting.

Reply:
It's the standard across the project, as odd as it looks.

Reply:
That's awful.